src/template-parameters.json

```diff
@@ -460,16 +460,6 @@
         "simpleValueType": true,
         "type": "TEXT",
         "enablingConditions": [
-          {
```

Comment:
Was this removal intended? `clickedObjectIDs` for example needs the `objectIDs` field here, see https://www.algolia.com/doc/api-reference/api-methods/clicked-object-ids/?client=javascript

Reply:
It was. I checked the API and was trying to limit the confusion. We only need revenue info on the conversion events (can you think of why we might need it on the non-conversion events?)  so I thought to remove it.

Reply:
Don't we pass `objectIDs` to click events? i.e. the object that was clicked? https://www.algolia.com/doc/api-reference/api-methods/clicked-object-ids/?client=javascript#examples

Similarly with view events, the objects that were viewed?

Reply:
`objectIDs` yes, `objectData` no.

Reply:
You've removed these from under `objectIDs` though, unless I'm reading it wrong.